models.rb

```diff
@@ -114,6 +125,11 @@ def validate
     super
     errors.add(:name, 'cannot be empty') if !name || name.empty?
   end
+
+  def reassign_resources
+    DbResourcesProject.where(db_resource_id: id).delete
+    update(active: false)
```

Comment:
This caused me some confusion, since it isn't really reassigning resources, but just doing some extra things on delete. It should be renamed of the logic moved into the ``delete`` method for this model